src/app/components/settings/settings.ui

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk" version="4.0" />
+  <requires lib="libadwaita" version="1.0" />
+  <template class="SettingsWindow" parent="AdwPreferencesWindow">
+    <property name="default-width">600</property>
+    <property name="hide-on-close">1</property>
+    <child>
+      <object class="AdwPreferencesPage">
+        <child>
+          <object class="AdwPreferencesGroup">
+            <property name="title" translatable="yes" comments="Header for a group of preference items regarding audio">Audio</property>
+            <child>
+              <object class="AdwComboRow" id="audio_backend">
+                <property name="title" translatable="yes" comments="Title for an item in preferences">Audio Backend</property>
+                <property name="model">
+                  <object class="GtkStringList">
+                    <items>
+                      <item>PulseAudio</item>
+                      <item>ALSA</item>
+                    </items>
+                  </object>
+                </property>
+              </object>
+            </child>
+            <child>
+              <object class="AdwActionRow">
+                <property name="title" translatable="yes" comments="Title for an item in preferences">ALSA Device</property>
+                <property name="subtitle" translatable="yes" comments="Description for the item (ALSA Device) in preferences">Applied only if audio backend is ALSA</property>
+                <child>
+                  <object class="GtkEntry" id="alsa_device"></object>
```

Comment:
```suggestion
                  <object class="GtkEntry" id="alsa_device">
                      <property name="valign">center</property>
                  </object>
```
Otherwise it looks weirdly stretched.